addons/account/wizard/account_automatic_entry_wizard_views.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="account_automatic_entry_wizard_form" model="ir.ui.view">
+            <field name="name">account.automatic.entry.wizard.form</field>
+            <field name="model">account.automatic.entry.wizard</field>
+            <field name="arch" type="xml">
+                <form>
+                    <field name="account_type" invisible="1"/>
+                    <field name="company_id" invisible="1"/>
+                    <field name="display_currency_helper" invisible="1"/>
+                    <div attrs="{'invisible': [('display_currency_helper', '=', False)]}"  class="alert alert-info text-center" role="status">
+                        The selected destination account is set to use a specific currency. Every entry transferred to it will be converted into this currency, causing
+                        the loss of any pre-existing foreign currency amount.
+                    </div>
+                    <field name="action" invisible="context.get('hide_automatic_options')" widget="radio" options="{'horizontal': true}"/>
+                    <group>
+                        <group attrs="{'invisible': [('action', '!=', 'accrual')]}">
+                            <field name="date" string="Recognition Date"/>
+                            <field name="expense_accrual_account" string="Accrued Account"
+                                   attrs="{'invisible': [('account_type', '!=', 'expense')], 'required': [('account_type', '=', 'expense'), ('action', '=', 'accrual')]}"/>
+                            <field name="revenue_accrual_account" string="Accrued Account"
+                                   attrs="{'invisible': [('account_type', '!=', 'income')], 'required': [('account_type', '=', 'income'), ('action', '=', 'accrual')]}"/>
+                        </group>
+                        <group attrs="{'invisible': [('action', '!=', 'transfer')]}">
+                            <field name="destination_account_id" attrs="{'required': [('action', '=', 'transfer')]}" domain="[('company_id', '=', company_id)]"/>
```

Comment:
This field isn't at the same place as the account in accrual mode, it's a bit confusing. Switch it with the date here, or in accrual.